aml_module/module_spec.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 description: This module implements the spectral residual anomaly detection algorithm for time-series.
 inputs:
   - name: Dataset
-    type: DataFrameDirectory
+    type: AnyDirectory
```

Comment:
Use multiple data type.
type:
  - AnyDirectory
  - DataFrameDirectory

Reply:
Or we can keep DataFrameDirectory, since AnyDirectory could link to DataFrameDirectory